### ISSUE TITLE
sendbid - Possible to specify account?

### ISSUE THREAD

**Issue by wi-ski** [handshake-org/hsd#82]
Hi there - love the project!

TL;DR - Reading through the source code and tinkering, is it possible to `sendbid` and specify the account to send the bid from?

Edit: More details.  I'm using `hs-client` to `sendbid`s. Looks like:
```js
      const bid = await walletClient.execute("sendbid", [
        newName,
        bidAmount,
        lockupAmount,
      ]);
```
In the source for `hsd` - this repo - the rpc module doesn't appear to accept any more params (like the account name) for `sendbid`.

https://github.com/handshake-org/hsd/blob/e0d57754e258ea8ea4a3d705fcc4e6ce6fed13f7/lib/wallet/rpc.js#L1896-L1918

_However..._

The interface for the wallet seems to expect the account as an optional param - see: https://github.com/handshake-org/hsd/blob/592b276077652fe0f3f09d1ed2bf49bbd4262ef4/lib/wallet/wallet.js#L1668-L1673

:male_detective: If that's the whole picture - whats the best way to specify a bid's account?

Would a patch be necessary/helpful? It would have to be across two libs I suppose... `hsd` and `hs-client`.

**Comment by hschoenburg** [comment]
Here is an example from the hsw-cli tool that might be helpful
https://github.com/handshake-org/hs-client/blob/master/bin/hsw-cli#L49

**Comment by wi-ski** [comment]
Thank you for the pointer :) What specifically were you trying to illustrate?

Apologies if I am being a complete idiot :pray:

**Comment by wi-ski** [comment]
This PR: https://github.com/handshake-org/hsd/pull/86 does stuff. Like, I can pass the `accountIndex` as a param.

It doesn't _feel_ right honestly.  Like, exposing the notion of account indices in the hs-client? Halp me.

**Comment by wi-ski** [comment]
![Alt Text](https://media.giphy.com/media/vFKqnCdLPNOKc/giphy.gif)

**Comment by pinheadmz** [comment]
closed by https://github.com/handshake-org/hsd/pull/233
